### ISSUE TITLE
hard limit for posting files associated w/ item

### ISSUE THREAD

**Issue by jordansread** [DOI-USGS/sbtools#187]
Looks like SB now has a limit for how many files can be associated w/ an item (100). if you try to post the 101 file, you get a really nasty error that doesn't tell you anything:
![image](https://cloud.githubusercontent.com/assets/2349007/13856042/62fef66a-ec41-11e5-83ba-929fc9ed17b2.png)

I am not suggesting the package should do much about this, but something to be aware of and maybe we should talk to SB team about returning an error message and code..?

**Comment by lawinslow** [comment]
Ha, I think they added that after our earlier, ~3k attached files, abuses of SB.

**Comment by jread-usgs** [comment]
nasty hobbitz

**Comment by sckott** [comment]
Do we know the number is 100 for sure?

**Comment by jread-usgs** [comment]
That is what I hit before getting this goofy fail every time I tried another file. I haven't asked them to confirm though

**Comment by lawinslow** [comment]
I've added a warning when posting large lists of files. Not a lot more I can do on the client side.

**Comment by aappling-usgs** [comment]
@jread-usgs - here's the associated issue you mentioned. Your notes from slack on the current experience from sbtools user perspective, posting many files 1 at a time:
> posting the 101st file to an item gives you a totally cryptic error from SB:
```
Error:
```
i like the idea of catching the cryptic error right where it happens and providing an interpretive message, e.g., "this error can occur if you try to post more than 100 files to a single item"

**Comment by jread-usgs** [comment]
Note the UI throws an error on save, but does let you upload the 101st file: 
![image](https://user-images.githubusercontent.com/2349007/31239583-3606fe32-a9c3-11e7-8c89-359d3a565956.png)

**Comment by aappling-usgs** [comment]
Goofy. Can you upload the 102nd in the same way?

**Comment by jread-usgs** [comment]
yes, same deal. You can upload 101 and 102, but the save is what fails
